app/src/components/GrantList.vue

```diff
@@ -89,12 +88,31 @@ export default defineComponent({
   setup(props) {
     const { addToCart, isInCart, removeFromCart } = useCartStore();
 
+    const sortedGrants = ref<Grant[]>([]);
```

Comment:
Having this logic within setup which also relies on the `useSortedGrants` composition function is a bit unconventional/confusing IMO. It's easier to read and reason about when composition functions are standalone methods that can be directly returned in the setup function, e.g. `setup() { return { ...useSortedGrants() } }`.

My preference would be to move the relevant portions of code from the `setup` method into the existed `useSortedGrants` composition function

Reply:
I've reverted this change 👍